static/app/components/events/eventStatisticalDetector/eventAffectedTransactions.tsx

```diff
@@ -257,6 +352,54 @@ function EventAffectedTransactionsInner({
   );
 }
 
+function findExamplePair(
+  examples: string[],
+  breakpointIndex,
```

Comment:
Wdyt about adding the breakpoint as the argument and doing the findIndex inside the body of this fn?

Reply:
Im tempted to leave it outside since then we'd have to pass the timestamps and breakpoint as an argument and I'd rather keep this function simple